peripherals-yaml/example.yaml

```diff
@@ -29,6 +29,9 @@ info:
   version: 0.1.0
   'x-foo': 'x-bar'
 
+imports:
```

Comment:
It looks like you moved this file, but we still require the original. Can you copy this file back and make sure `peripherals/example.yaml` does not include the `imports`?